build.config

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+CAKE_VERSION="0.38.4"
+DOTNET_VERSION="3.1.403"
```

Comment:
Is this a shell script? Shouldn't the extension be `.sh`?

Reply:
This is just a file read by build.sh and build.ps1